presto-main/src/main/java/io/prestosql/operator/OperatorStats.java

```diff
@@ -504,6 +518,8 @@ public OperatorStats add(Iterable<OperatorStats> operators)
                 blockedReason = operator.getBlockedReason();
             }
 
+            dynamicFilterSplitsProcessed += operator.getDynamicFilterSplitsProcessed();
```

Comment:
move this above `physicalWrittenDataSize += operator.getPhysicalWrittenDataSize().toBytes();`